lib/Analytics/AnalyticsDatasource.php

```diff
@@ -172,13 +176,69 @@ public function readData($option): array {
 		}
 		unset($rows);
 
-		return [
-			'header' => $header,
-			'dimensions' => array_slice($header, 0, count($header) - 1),
-			'data' => $data,
-			//'rawdata' => $data,
-			'error' => 0,
-		];
+		return ['header' => $header, 'dimensions' => array_slice($header, 0, count($header) - 1), 'data' => $data, //'rawdata' => $data,
+			'error' => 0,];
```

Comment:
Nitpick-as-a-Service: indentation looks odd. Would be better in the previous style.

Reply:
let me check which code styler setting messed this up

Reply:
saw to late. but a follow up pr is welcomed.